sklearn/tree/tree.py

```diff
@@ -26,21 +26,16 @@
 import numpy as np
 from scipy.sparse import issparse
 
-from ..base import BaseEstimator
-from ..base import ClassifierMixin
-from ..base import RegressorMixin
+from ..base \
```

Comment:
Please refrain from making changes unless there is a very strong reason behind it. This adds noise to the diff and this unnecessary messiness is quite annoying as a reviewer.